de-lib/src/main/java/org/iplantc/de/client/models/apps/App.java

```diff
@@ -149,4 +149,10 @@
 
     @PropertyName("job_stats")
     AppStats getAppStats();
+
+    AppExtra getExtra();
+    void setExtra(AppExtra extra);
+
+    AppDoc getDocumentation();
+    void setDocumentation(AppDoc appDoc);
```

Comment:
👍 
This is slightly confusing later, when we have to access `app.documentation.documentation` to get to the doc text.
Would it look better to use `app.docsinfo.documentation` / `getDocsInfo` / `setDocsInfo`, or something like that?

Reply:
Either way, 👍 for bundling this with the other app info.

Reply:
🤔 The keys back from the service are 
```
documentation: {
    app_id: "someId", 
    documentation: "myDocs",
    ...
}
```
So I could change the function name here in `App.java` to `getDocsInfo` and add the `@PropertyName` annotation to properly access `documentation`, but in javascript, it will still be `app.documentation.documentation` I thought because those are the real keys.

Reply:
Oh, oops. I didn't realize this already matched the service response. I thought we were using a different endpoint for docs and just bundling in the UI for convenience.

In that case, let's leave it this way so they match.